packages/ui-app/src/types.d.ts

```diff
@@ -25,3 +25,7 @@ export type BaseContext = {
     }
   }
 };
+
+export type SUIEvent = {
+  value: any
```

Comment:
Don't use `any` unless there is no option. Specify or use standard React events. (They are typed)